.github/workflows/dashboards-observability-test-and-build-workflow.yml

```diff
@@ -15,8 +15,10 @@ jobs:
     runs-on: ubuntu-latest
 
     steps:
-      - name: Checkout Plugin
-        uses: actions/checkout@v1
+      # Enable longer filenames for windows
+      - name: Enable longer filenames
+        if: ${{ matrix.os == 'windows-latest' }}
+        run: git config --system core.longpaths true
```

Comment:
sorry just realized this is not necessary because we are not running windows ci on 2.2 see line 15

Reply:
Got it will remove, thanks for following up.